### PR TITLE
Fix po files

### PR DESCRIPTION
**Pull request by pnax**
Add missing `.` in some msgid and update the language value in sv file.

### CODE REVIEW

File: share/da.po [hunk 1]
```diff
@@ -12,7 +12,7 @@ msgstr ""
 msgid "Print version information and exit."
 msgstr "Udskriv version og afbryd."
 
-msgid "The minimum severity level to display"
+msgid "The minimum severity level to display."
```
Comment (matsduf) [inline share/da.po:15]: That is not correct msgid from the code.

File: share/sv.po [hunk 2]
```diff
@@ -12,7 +12,7 @@ msgstr ""
 msgid "Print version information and exit."
 msgstr "Skriv ut versionsinformation och avsluta."
 
-msgid "The minimum severity level to display"
+msgid "The minimum severity level to display."
```
Comment (matsduf) [inline share/sv.po:15]: This is not correct msgid from the code.

**Review by matsduf — CHANGES_REQUESTED** [review]
The msgid should be updated by first doing any updates of the Perl module and then generate new PO files by e.g. "make da.po". By doing that, msgids will be correct.

**Comment by mattias-p** [comment]
It's important that the msgid is identical to the string given to the `__(...)` function (or one of its friends). That's one of the core principle of how the gettext system works.

But it looks to me like we've forgotten to ask for new translations for CLI. I'll make a PR that fixes the formatting of the PO files to make life easier for our translators.

File: share/da.po (hunk 1 above)
Comment (pnax) [reply share/da.po:15]: Yes indeed, I thought the rest of the msgid was wrapped on a second line.

File: share/sv.po (hunk 2 above)
Comment (pnax) [reply share/sv.po:15]: Yes, my mistake. And thanks for the tip with the make command ;)

**Comment by pnax** [comment]
> It's important that the msgid is identical to the string given to the __(...) function (or one of its friends). That's one of the core principle of how the gettext system works.

Yes, I thought this was just a missing dot. I'll be more careful with my greps.

I am closing this PR because this is not the proper way of updating po files, there is a make command for that.

**Pull request closed without merging**